R-package/README.md

```diff
@@ -0,0 +1,561 @@
+
+<br>
+
+**UPDATE 26-07-2018**: A [Singularity image file](http://mlampros.github.io/2018/07/26/singularity_containers/) is available in case that someone intends to run *RGF* on Ubuntu Linux (locally or in a cloud instance) with all package requirements pre-installed. This allows the user to utilize the *RGF* package without having to spend time on the installation process.
+
+<br>
+
+**References:**
+
+[Rie Johnson and Tong Zhang, Learning Nonlinear Functions Using Regularized Greedy Forest](https://arxiv.org/abs/1109.0887)
+
+[RGF](https://github.com/RGF-team/rgf/tree/master/RGF)
+
+[FastRGF](https://github.com/RGF-team/rgf/tree/master/FastRGF)
+
+[python-package](https://github.com/RGF-team/rgf/tree/master/python-package)
+
+
+<br>
+
+### **System Requirements**
+
+<br>
+
+* Python (2.7 or >= 3.4)
+
+
+<br>
+
+All modules should be installed in the default python configuration (the configuration that the R-session displays as default), otherwise errors will occur during the *RGF* package installation (**reticulate::py_discover_config()** might be useful here). 
+
+<br>
+
+#### **Debian/Ubuntu/Fedora**    [ installed / tested on Python 2.7 ]
+
+<br>
+
+First install / upgrade the dependencies,
+
+<br>
+
+```R
+sudo pip install --upgrade pip setuptools
+
+sudo pip install -U numpy
+
+sudo pip install --upgrade scipy
+
+sudo pip install -U scikit-learn
+```
+
+<br>
+
+Then, download the *rgf-python* package and install it using the following commands,
+
+<br>
+
+
+```R
+
+git clone https://github.com/RGF-team/rgf.git
+
+cd rgf/python-package
+
+sudo python setup.py install
+
+```
+
+<br>
+
+*FastRGF* will be installed successfully only if gcc >= 5.0.
+
+<br><br>
+
+
+
+#### **Macintosh OSX**            [ installed / tested on Python >= 3.4 ]
+
+<br>
+
+Upgrade python to version 3 using, 
+
+
+```R
+
+brew upgrade python
+
+```
+<br>
+
+Then install the dependencies for *RGF* and *FastRGF*
+
+<br>
+
+```R
+sudo pip3 install --upgrade setuptools
+
+sudo pip3 install -U numpy
+
+sudo pip3 install --upgrade scipy
+
+sudo pip3 install -U scikit-learn
+```
+<br>
+
+
+The *FastRGF* module requires a gcc >= 8.0. To install *gcc-8* (or the most recent gcc) with brew follow the next steps,
+
+<br>
+
+```R
+
+# before running the following commands make sure that the most recent Apple command line tools for Xcode are installed
+
+brew update
+
+brew upgrade
+
+brew info gcc
+
+brew install gcc
+
+brew cleanup
+
+```
+<br>
+
+After the newest gcc version is installed the user should navigate to */usr/local/bin* and if a *gcc* file exists (symbolic link) then the user should delete it. Then the user should
+run the following command,
+
+<br>
+
+```R
+
+sudo ln -s /usr/local/bin/gcc-8 /usr/local/bin/gcc
+
+```
+
+<br>
+
+The user should then verify that the gcc has been updated using,
+
+<br>
+
+```R
+
+gcc -v
+
+which gcc
+
+```
+
+<br>
+
+After the new gcc is installed the user should continue with the installation of *rgf-python*,
```

Comment:
Many thanks for your fixes!

But can you please re-generate `Rd` files, so we might go further?

In addition, last minor fix: `rgf_python`, not `rgf-python`.